.github/workflows/superset-applitool-cypress.yml

```diff
@@ -59,8 +64,8 @@ jobs:
         uses: ./.github/actions/cached-dependencies
         with:
           run: testdata
-      - name: Set up Node.js
-        uses: actions/setup-node@v3.1.1
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
```

Comment:
Same here - I'd rather use `actions/setup-node@v3` as it uses Node 16 (https://github.com/actions/setup-node/releases/tag/v3.0.0) - any reason why we're tagging `v2`?

Reply:
I think this was just a copy and paste error on my side and also because most of the other actions are using v2. Let me do this in the follow-up as well so I can test all these changes you suggested locally.